### ISSUE TITLE
cache for pseudobulk same as for single cell?

### ISSUE THREAD

**Issue by stemangiola** [stemangiola/CuratedAtlasQueryR#144]
@myushen I noticed that if I query pseudobulk

```r
my_metadata |>
     get_pseudobulk(assays = "counts")
```

the data is stored in the wrong cache?

```r
Downloading https://object-store.rc.nectar.org.au/v1/AUTH_06d6e008e3e642da99d806ba3ea629c5/pseudobulk-0.1.0/original/GSE122960/se.rds to /Users/mangiola.s/Library/Caches/org.R-project.R/R/CuratedAtlasQueryR/0.2.1/original/GSE122960/se.rds
```

I think we should use `pseudobulk-0.1.0` for the cache , not  `0.2.1`

**Comment by myushen** [comment]
good point. this is associated with the cache directory structure strategy implementation. we might need to think about if we want to adapt the same structure strategy as the future cloud.
cellxgene
- date (yy-mm-dd)
- - original
- - cpm
- another date (yy-mm-dd)
-- original
-- cpm

fibrosis
- date (yy-mm-dd)
- - original
- - cpm

pseudobulk
- date (yy-mm-dd)
- - original
- - quantile_normalised

**Comment by stemangiola** [comment]
How about this hierarchy

cellxgene

- date (yy-mm-dd)
-- original
-- cpm
-- pseudobulk
- another date (yy-mm-dd)
-- original
-- cpm
-- pseudobulk

fibrosis

...
